testing/stubtestclient/main.go

```diff
@@ -56,6 +57,7 @@ func init() {
 	flag.StringVar(&experiment, "experiment", "exp1", "experiment")
 	flag.StringVar(&variation, "variation", "var1", "variation")
 	flag.StringVar(&installerType, "installer_type", "full", "installer_type")
+	flag.StringVar(&installerType, "visit_id", "vid", "visit_id")
```

Comment:
Should it be `&visitId` as the first argument?

Reply:
Thanks, that was a typo!